README.md

```diff
@@ -105,9 +105,17 @@ You can then create a PR from the release branch. Once your PR has been approved
 and merged, the CI pipeline will automatically publish packages to NPM, tag the
 release, and create a release in Github.
 
-If the publishing, open a new PR with any fixes and merge your changes. CI will
+If the publishing fails, open a new PR with any fixes and merge your changes. CI will
 attempt to republish the previous release that failed.
 
+### Release Notes
+
+Releases will be populated with release notes automatically based on the contents of
+the PR description. The release notes are generated using the contents of the `Summary`
```

Comment:
For extra clarity, maybe say "the release PR description?"

Reply:
Good thought! Updated with https://github.com/Vertexvis/vertex-web-sdk/pull/588/commits/d534e593941adee23fb5a36ca87a3f82ee24154a